doc/qbittorrent.1.md

```diff
@@ -0,0 +1,36 @@
+% QBITTORRENT(1) Bittorrent client written in C++ / Qt
+% Christophe Dumez <chris@qbittorrent.org>
+% January 16th 2010
+
+# NAME
+qBittorrent - a Bittorrent client written in C++ / Qt
+
+
+# SYNOPSIS
+**qbittorrent** `[--no-splash] [--webui-port=x] [TORRENT_FILE | URL]...`
+
+**qbittorrent** `--help`
+
+**qbittorrent** `--version`
+
+
+# DESCRIPTION
+qBittorrent is an advanced Bittorrent client written in C++ / Qt,
+using the libtorrent-rasterbar library by Arvid Norberg. qBittorrent is similar to uTorrent.
```

Comment:
Missing bold text for `qbittorrent` and `libtorrent-rasterbar`.

Reply:
Done.